django/thunderstore/api/cyberstorm/views/dynamic_html.py

```diff
@@ -0,0 +1,32 @@
+from django.http import Http404
+from django.utils.html import escapejs
+from rest_framework import serializers
+from rest_framework.generics import RetrieveAPIView
+
+from thunderstore.api.utils import CyberstormAutoSchemaMixin
+from thunderstore.frontend.models import DynamicHTML, DynamicPlacement
+
+
+class CyberstormDynamicHTMLResponseSerializer(serializers.Serializer):
+    dynamic_htmls = serializers.ListField(child=serializers.CharField())
+
+
+class DynamicHTMLAPIView(CyberstormAutoSchemaMixin, RetrieveAPIView):
+    """
+    Return Cyberstorm dynamic html placement contents as prerendered HTML.
+    """
+
+    serializer_class = CyberstormDynamicHTMLResponseSerializer
+
+    def get_object(self):
+        if (
+            self.kwargs["placement"].startswith("cyberstorm_")
+            and self.kwargs["placement"] in DynamicPlacement.options()
+        ):
```

Comment:
no need to check for the cyberstorm prefix, just check if the kwarg is a valid option. also invert the if clause.